src/isPalindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(inputString){
```

Comment:
Consider removing the type name from the variable - `input` is sufficient to communicate the purpose of the parameter for this function, and `String` may not be true (for example, if invalid input is supplied to the function).